app/mailers/rent_mailer.rb

```diff
@@ -0,0 +1,13 @@
+class RentMailer < ApplicationMailer
+  # Subject can be set in your I18n file at config/locales/en.yml
+  # with the following lookup:
+  #
+  #   en.rent_mailer.rent_created.subject
+  #
+  def rent_created
+    @rent = Rent.last
+    @book_rented = Book.find(@rent.book_id)
+    @user_renting = User.find(@rent.user_id)
```

Comment:
For both relations, you can do @rent.book, or @rent.user to get the instance of the relation. It's easier and in some cases it saves some queries to the db